### ISSUE TITLE
Too many "trace" packages

### ISSUE THREAD

**Issue by bogdandrutu** [open-telemetry/opentelemetry-go#1014]
```golang
import "go.opentelemetry.io/otel/api/trace"
import "go.opentelemetry.io/otel/sdk/trace"
import "go.opentelemetry.io/otel/sdk/export/trace"
```

It is very hard to work with all of these. Would suggest maybe use different names, maybe consider:
`sdktrace` for "go.opentelemetry.io/otel/sdk/sdktrace"
`exptrace` for "go.opentelemetry.io/otel/sdk/export/exptrace"

**Comment by jmacd** [comment]
We discussed this in the SIG, and think that probably users should rename these imports as needed. Only SDK maintainers and exporter maintainers will have this trouble.  Also note, e.g., in the metrics code we remap go.opentelemetry.io/otel/sdk/export/metric as

```
import export "go.opentelemetry.io/otel/sdk/export/metric"
```

**Comment by bogdandrutu** [comment]
That is not true, I have this problem in my main file because I need to configure SDK/API and exporter.

**Comment by bogdandrutu** [comment]
Also in tests because I need to use the test exporter

**Comment by MrAlias** [comment]
This is partially related to #964. I think we can try to do the same in the SDK and SDK exporter package.

**Comment by MrAlias** [comment]
Closing as there is not a desire to resolve across the SDK or API packages. The export package will likely be addressed in #1380
